source/docs/articles/going-live.md

```diff
@@ -5,204 +5,56 @@ category:
+&nbsp;&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Adjust TTL Value to 120s](https://en.wikipedia.org/wiki/Time_to_live#DNS_records)<br>
 
 ## Schedule Backups
 
-Ensure that your Live environment content is protected by scheduling daily and weekly backups. That way you've got a fallback in case you want to revert your site content.
-
-<table class=table>
-<tbody>
-		<tr>
-			<th class="complete">Complete</th>
-			<th class="action">Action</th>
-			<th class="help">Help</th>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Schedule backups</td>
-			<td class="help"><a href="/docs/articles/sites/backups/backup-creation">Creating a backup</a></td>
-		</tr>
-	</tbody>
-	</table>
-
-## Add Domain to the Live Environment
-
-Now that the site is on a paid plan, you can associate your domain with the Live environment. This tells Pantheon where to send site traffic.
-
-<table class=table>
-<tbody>
-		<tr>
-			<th class="complete">Complete</th>
-			<th class="action">Action</th>
-			<th class="help">Help</th>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Add domain(s) to Live environment</td>
-			<td class="help"><a href="/docs/articles/sites/domains/adding-a-domain-to-a-site-environment/">Adding a domain to a site environment</a><p>
-      <a href="/docs/articles/sites/domains/">Domains and SSL Tool</a></td>
-		</tr>
-	</tbody>
-	</table>
+Ensure that your content is protected by scheduling daily and weekly backups. That way you've got a fallback in case you want to revert your site content. Backups need to be run separately for each environment (Dev, Test, and Live). For detailed instructions, see <a href="/docs/articles/sites/backups/backup-creation">Creating a backup</a>.
+
+## Monitoring Services
+
+Consider enabling [New Relic](/docs/articles/sites/newrelic/new-relic-performance-analysis/) monitoring to non-intrusively track the performance of your site; this is one of the first places Pantheon will look when there are performance concerns.
+
+[Pingodm](https://www.pingdom.com/) can also be used as an additional monitoring service to create site alerts for non-cached pages.
+
+##  Redirects
+
+Setup redirects for possible URL changes as a result of going live. Have plan to monitor and mitigate 404 Page not found errors post-launch to retain search rankings. For more information, see [Redirecting Incoming Requests](/docs/articles/sites/code/redirect-incoming-requests/)
+<div class="alert alert-danger" role="alert">
+<strong>Warning</strong>: For best results, do <strong>not</strong> rely global redirect logic within the application configuration (e.g. wp-config.php or settings.php). The <a href="https://www.drupal.org/project/redirect" target="blank">Redirect module</a> is recommended. </div>

```

Comment:
@suzannealdrich is this what you meant by global redirects? Are there any WP redirect plugins we can recommend? Did I leave anything out here? @ari @JessiFischer